infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,28 @@
+import * as cdk from "@aws-cdk/core";
+import * as ecs from "@aws-cdk/aws-ecs";
+import * as ecsPatterns from "@aws-cdk/aws-ecs-patterns";
+
+export class InfrastructureStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    new ecsPatterns.ApplicationLoadBalancedFargateService(this, `${id}-ecs`, {
+      ...props,
+      memoryLimitMiB: 1024,
+      cpu: 512,
+      taskImageOptions: {
+        image: ecs.ContainerImage.fromAsset("../", {
+          exclude: ["infrastructure/*"],
+        }),
+        // here's how we'd grab the image from docker hub:
+        // image: ecs.ContainerImage.fromRegistry(
+        //   "joshprzybyszewski/cribbage:latest"
+        // ),
+        containerPort: 8080,
+        environment: {
+          CRIBBAGE_DB: "memory",
+        },
```

Comment:
@joshprzybyszewski I was being kinda dumb; now I'm being less dumb. The global config package will map this env var to the `db` command line arg (which to me is a bit confusing; it's a bit hard to keep track of where configuration is coming from, but I'll get over it :sunglasses:).